runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/ModelerPanels/RunSavePanel.tsx

```diff
@@ -21,18 +24,39 @@ interface RunSavePanelProps {
 const RunSavePanel: FC<RunSavePanelProps> = ({
     onRunClick,
     onSave,
-    process
+    process,
 }) => {
     const { translate } = useTranslations();
     const { isSaveDisabled, errors } = useSelector(
-        state => state.process.modeler
+        (state) => state.process.modeler
     );
     const getTooltip = () => {
-        const elementsWithErrors = errors.map(error => error.elementName);
-        if (elementsWithErrors.length > 0) {
-            return translate('Process.MainView.Tooltip.Save.Errors', {
-                errors: elementsWithErrors.join(', ')
-            });
+        const elementsWithFromErrors = errors
+            .map((error) =>
+                error.type === ModelerErrorType.FORM_ERROR
+                    ? error.elementName
+                    : null
+            )
+            .filter((element) => element !== null);
+
+        const elementsWithConnectionErrors = errors
+            .map((error) =>
+                error.type === ModelerErrorType.CONNECTION_ERROR
+                    ? error.elementName
+                    : null
+            )
+            .filter((element) => element !== null);
```

Comment:
`reduce` will be better for this use case (single loop over the items instead of 2)